lib/planet.rb

```diff
@@ -0,0 +1,5 @@
+class Planet
+  def initialize(name)
+    @name = name
```

Comment:
Is this what we want to name the attribute?

Reply:
yes